power-sequencer/main.cpp

```diff
@@ -54,30 +52,25 @@ int main(int argc, char** argv)
     auto bus = sdbusplus::bus::new_default();
     bus.attach_event(event.get(), SD_EVENT_PRIORITY_NORMAL);
 
+#ifdef UCD90160_DEVICE_ACCESS
```

Comment:
UCD90160_DEVICE_ACCESS is not used to check UCD or mihawk. You need to create another macro.

Reply:
@alvintpwang I will fix it.